projects.rst

```diff
@@ -81,128 +108,40 @@ like a particular hardware or software platform,
-
-Documentation
-=============
-
-While each project is documented individually,
-some information touches multiple projects
-and is tracked elsewhere.
-
-Nengo documentation
--------------------
-
-You're reading the general Nengo documentation right now!
-It contains general information about
-projects in the Nengo ecosystem
-and how these projects are developed.
-
-==================   ========================================
-Primary maintainer   Trevor Bekolay <tbekolay@gmail.com>
-Github               https://github.com/nengo/nengo.github.io
-==================   ========================================
-
-Enhancement proposals
----------------------
-
-Nengo Enhancement Proposals (NEPs) are proposals
-that touch multiple projects in the Nengo ecosystem,
-or represent a large change to one important project.
-
-==================   ==============================================
-Primary maintainer   Trevor Bekolay <tbekolay@gmail.com>
-Github               https://github.com/nengo/enhancement_proposals
-==================   ==============================================
-
-In development
-==============
-
-These projects are still in the early stages of development.
-If you're interested in helping out,
-please contact the maintainer!
-
-Nengo SPA
----------
-
-The Semantic Pointer Architecture (SPA)
-uses Nengo to build large cognitive models.
-
-==================   ==================================
-Primary maintainer   Jan Gosmann <jan@hyper-world.de>
-Github               https://github.com/nengo/nengo_spa
-==================   ==================================
-
-Detailed neuron models
-----------------------
-
-This project enables Nengo use neuron models
-simulated with `NEURON <http://www.neuron.yale.edu/neuron/>`_.
-
-==================   ===============================================
-Primary maintainer   Jan Gosmann <jan@hyper-world.de>
-Github               https://github.com/nengo/nengo_detailed_neurons
-==================   ===============================================
-
-Nengo benchmarks
-----------------
-
-Models and utilities for benchmarking
-different Nengo versions and backends.
-
-==================   =========================================
-Primary maintainer   Terry Stewart <terry.stewart@gmail.com>
-Github               https://github.com/nengo/nengo_benchmarks
-==================   =========================================
+.. project:: Nengo OpenCL
+   :repo: nengo/nengo_ocl
+   :maintainer: `Eric Hunsberger <erichuns@gmail.com>`_
+   :pypi: nengo_ocl
+
+   Nengo OpenCL uses the `OpenCL framework <https://www.khronos.org/opencl/>`_
+   to run Nengo models on GPUs and other platforms.
+   Most models run significantly faster with Nengo OpenCL.
+
+.. project:: Nengo deep learning
+   :repo: nengo/nengo_deeplearning
+   :maintainer: `Daniel Rasmussen <dhrsmss@gmail.com>`_
+
+   Nengo deep learning simulates Nengo models using
+   the `TensorFlow <https://www.tensorflow.org/>`_ library
```

Comment:
It'll all be switching over to Tensorflow for the first official "release" (maybe in a month or so, rough estimate).